### PR TITLE
iOS feature: Hardware wallet settings.

### PR DESCRIPTION
**Pull request by liuzjalex**
## What does this implement/fix? Explain your changes.
Implementation of hardware wallet settings.

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
…

## Pull request type
 
_Put an `x` in the boxes that apply_
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
iPhone xs max iOS 14.3

## Any other comments?
…

### CODE REVIEW

File: ios/CustomCode/Modules/Hardware/Controller/OKMatchingInCirclesViewController.m [hunk 1]
```diff
@@ -197,7 +197,7 @@ - (void)subscribeComplete
         OKDeviceInfoModel *model = [OKDeviceInfoModel mj_objectWithKeyValues:jsonStr];
         kOKBlueManager.model = model;
         dispatch_async(dispatch_get_main_queue(), ^{
-            if ([model.initialized boolValue]) {
```
Comment (DevoutCoder) [inline ios/CustomCode/Modules/Hardware/Controller/OKMatchingInCirclesViewController.m:200]: Wrong change

File: ios/CustomCode/Modules/Hardware/Controller/OKMatchingInCirclesViewController.m [hunk 2]
```diff
@@ -197,7 +197,7 @@ - (void)subscribeComplete
...
+            if (model.initialized ) {
```
Comment (DevoutCoder) [inline ios/CustomCode/Modules/Hardware/Controller/OKMatchingInCirclesViewController.m:200]: Wrong change

Comment (liuzjalex) [reply ios/CustomCode/Modules/Hardware/Controller/OKMatchingInCirclesViewController.m:200]: 模型改过了现在是bool类型。

**Review by DevoutCoder — APPROVED** [review]
no problem

**Pull request merged**